### PR TITLE
Disables Fly & Squid characters

### PR DESCRIPTION
**Pull request by Crossedfall**
## About The Pull Request

Fly/squid characters will no longer be selectable races for new characters. ~~Pre-existing characters can still be played like normal~~

I also sorted the species alphabetically and grouped them based on their state. 

(Note: Removed (mil) synths since Kapu forgot to do so in their pr #5623)

## Why It's Good For The Game

Both Flies and Squids are severely underutilized and have been lacking in updates. This prevents them from bloating the current species list while still allowing people who primarily play as either species to continue doing so for the time being.

## Changelog
:cl:
config: Squid & fly have been removed as selectable species
/:cl:

### CODE REVIEW

**Comment by Crossedfall** [comment]
**Notice:**  This is currently applied to the server, but I will leave the PR up for a bit on the off chance there's a proper blocking reason on why this shouldn't be applied long-term.

**Comment by PowerfulBacon** [comment]
I didnt even know fly people were on, I thought they got replaced by apids

File: config/game_options.txt [hunk 1]
```diff
@@ -478,38 +478,44 @@ SILICON_MAX_LAW_AMOUNT 12
...
 #ROUNDSTART_RACES silver_golem
-#ROUNDSTART_RACES uranium_golem
-#ROUNDSTART_RACES abductor
 #ROUNDSTART_RACES synth
-ROUNDSTART_RACES ipc
+#ROUNDSTART_RACES uranium_golem
 
 ## Races that are straight upgrades. If these are on expect powergamers to always pick them
+#ROUNDSTART_RACES agent
+#ROUNDSTART_RACES military_synth
```
Comment (Ivniinvi) [inline config/game_options.txt]:
```suggestion
```
milsynths are gone 🦀

Comment (Crossedfall) [reply config/game_options.txt]: Way to be late

**Review by ike709 — DISMISSED** [review]

**Comment by BrumalLake** [comment]
no real reason to remove them
they're 2 lines in a list you look at only during customization
hair lists and co are much longer

**Comment by Crossedfall** [comment]
> no real reason to remove them they're 2 lines in a list you look at only during customization hair lists and co are much longer

They're unmaintained and unused. Those are two fairly good reasons to remove them.

**Comment by Wilchenx** [comment]
what does "unmaintained" mean in this context

**Comment by DrMacCool** [comment]
It'll be a shame to see the two go. They were the only species that were somewhat 'gross', which is refreshing when compared to all the very aesthetically clean, dare I say 'cute', species we have. Hopefully someone, someday will embrace the weirdness that can come with alien species and give us the new era of crewmember that makes you feel weird being next to, and not because they're trying to cram their cutesy, owo-talk persona down your throat.

**Comment by DrMacCool** [comment]
> 
> 
> >
> 
> They're unmaintained and unused. 

I'll have you know I submitted a PR for squidpeople like a month ago.

**Comment by chizzy376** [comment]
Why not just.. maintain them instead of half-removing them

**Comment by Crossedfall** [comment]
> what does "unmaintained" mean in this context

Lacking support/interest from contributors, but mostly from maintainers.

> Why not just.. maintain them instead of half-removing them

Unless you're volunteering to solely maintain them, nobody wants to support them.

**Comment by Crossedfall** [comment]
Fun revelation. 

`ROUNDSTART_NO_HARD_CHECK` doesn't have any code associated with it on Bee or TG. It literally does nothing. Incredible.

File: config/game_options.txt [hunk 2]
```diff
@@ -478,38 +478,42 @@ SILICON_MAX_LAW_AMOUNT 12
...
-#ROUNDSTART_RACES pod
-#ROUNDSTART_RACES military_synth
-#ROUNDSTART_RACES agent
+#ROUNDSTART_RACES zombie
+
+## Roundstart no-reset races
+##-------------------------------------------------------------------------------------------
+## Races defined here will not cause existing characters to be reset to human if they currently have a non-roundstart species defined.
+ROUNDSTART_NO_HARD_CHECK squid
+ROUNDSTART_NO_HARD_CHECK fly
```
Comment (Crossedfall) [inline config/game_options.txt]:
```suggestion
```

**Review by Archanial — APPROVED** [review]

**Pull request merged**